# 
string query
---
std_msgs/Header header
int32 plan_result
int32 SUCCESS=1
int32 FAIL=0
socialrobot_actionlib/Action[] action_sequence
